romancal/regtest/test_skycell.py

```diff
@@ -0,0 +1,82 @@
+"""Unit tests for skycell WCS on global sky map"""
+
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+import romancal.skycell.skymap as sc
+
+# mark all tests in this module
+pytestmark = [pytest.mark.bigdata]
+
+
+def assert_allclose_lonlat(actual: np.ndarray, desired: np.ndarray, rtol=1e-7, atol=0):
+    assert_allclose(
+        np.array(actual) % 360, np.array(desired) % 360, rtol=rtol, atol=atol
+    )
+
+
+@pytest.mark.parametrize(
+    "name",
+    [
+        "000p86x61y68",
+        "045p86x29y34",
+        # north pole
+        "225p90x25y49",
+        # south pole
+        "135m90x40y46",
+    ],
+)
+def test_skycell_wcs_pixel_to_world(name):
+    skycell = sc.SkyCell.from_name(name)
+
+    wcs = skycell.wcs
+
+    # forward transform to radec corners
+    assert_allclose_lonlat(
+        np.array(
+            wcs(
+                *np.array(
+                    [
+                        (0.5, 0.5),
+                        (skycell.pixel_shape[0] + 0.5, 0.5),
+                        (skycell.pixel_shape[0] + 0.5, skycell.pixel_shape[1] + 0.5),
+                        (0.5, skycell.pixel_shape[1] + 0.5),
+                    ]
+                ).T
+            )
+        ).T,
+        skycell.radec_corners,
+        rtol=1e-7,
+    )
```

Comment:
Do these corners look a little peculiar?  I think I would have guessed (-0.5, 0.5) to (shape - 0.5, shape - 0.5).  Unless this is fits vs. python, in which case we probably want a reference file update?